crates/common/tedge_utils/Cargo.toml

```diff
@@ -11,20 +11,27 @@ description = "tedge_utils provide utilities for thin-edge.io components"
 # No features on by default
 default = []
 logging = ["tracing", "tracing-subscriber"]
+fs-notify = ["inotify"]
```

Comment:
Is only `inotify` an optional dependency? I was expecting to have also the other crates you added to implement that `fs-notify` features, i.e. `async-stream`, `strum_macros` and `try-traits`.